src/display/canvas.js

```diff
@@ -1503,8 +1510,12 @@ class CanvasGraphics {
     }
   }
 
-  checkSMaskState() {
-    const inSMaskMode = !!this.suspendedCtx;
+  inSMaskMode() {
+    return !!this.suspendedCtx;
+  }
+
+  checkSMaskState(fromRestore = false) {
```

Comment:
I might be wrong, but it appears that the `fromRestore` parameter is completely unused and could thus be removed.

Reply:
Given that we have a bunch abstract base-classes and interface definition, where any parameters aren't being used, the currently ESLint is configured to not check function/method definitions.

I can look into fixing that (with manually white-listing the appropriate cases), since I agree that this is less than ideal.